pkg/lang/ir/v1/system.go

```diff
@@ -229,44 +230,63 @@ func (g *generalGraph) compileExtraSource(root llb.State) (llb.State, error) {
 }
 
 func (g *generalGraph) compileLanguage(root llb.State) (llb.State, error) {
+	var wg sync.WaitGroup
```

Comment:
You don't need to use the goroutine and wait group here. The BuildKit LLB only builds the DAG graph and executes it later.

Reply:
Updated